### ISSUE TITLE
Why is there so much difference between the miou 0.78 I reproduced and the one in the paper?

### ISSUE THREAD

**Issue by Aurora6899** [weiguangzhao/BFANet#6]
[2025-07-03 05:04:24,799  INFO  train.py  line 145  1161623]  >>>>>>>>>>>>>>>> Start Evaluation >>>>>>>>>>>>>>>>
iter: 312/312 loss: 2.5875(3.8218) Accuracy 0.9521  
[2025-07-03 05:07:56,436  INFO  train.py  line 213  1161623]  epoch: 401/401, val loss: 3.8218,  time: 211.63504219055176s
[2025-07-03 05:07:56,437  INFO  train.py  line 232  1161623]  mIoU/mAcc/allAcc 0.7384/0.8243/0.9094.  
[2025-07-03 05:07:56,438  INFO  train.py  line 234  1161623]  best_mIoU: 0.7438.

**Comment by weiguangzhao** [comment]
Thanks for your attention. Our training setup involves using four 4090 GPUs for training, and our multiple runs have consistently achieved a best mIoU score of no less than 77.0. What is your training configuration? Have you made any changes to the training parameters?

**Comment by Aurora6899** [comment]
Thank you for your reply. The devices I'm using for reproduction are two NVIDIA GeForce RTX 3090 GPUs. I only modified the &nbsp;val_freq&nbsp; parameter to 1 during training. I didn't make any other changes to the training parameters!! I will further investigate the root cause of the problem.








 ----------Reply to Message----------
 On Thu, Jul 3, 2025 10:37 AM Weiguang ***@***.***&gt; wrote:

  weiguangzhao left a comment (weiguangzhao/BFANet#6)
 
Thanks for your attention. Our training setup involves using four 4090 GPUs for training, and our multiple runs have consistently achieved a best mIoU score of no less than 77.0. What is your training configuration? Have you made any changes to the training parameters?

**Comment by weiguangzhao** [comment]
I am concerned that using only two 3090 GPUs may result in a small batch size, which could lead to overfitting. I suggest using four GPUs for training. I will close this repository first. If there are any issues, feel free to reopen it and ask further questions.

**Comment by Aurora6899** [comment]
Thank you for your reply. I will use four NVIDIA GeForce RTX 4090 GPUs for reproduction and restore the original training parameter settings. Then I'll further investigate the cause of the problem.








 ----------Reply to Message----------
 On Thu, Jul 3, 2025 10:58 AM Weiguang ***@***.***&gt; wrote:

  weiguangzhao left a comment (weiguangzhao/BFANet#6)
 
I am concerned that using only two 3090 GPUs may result in a small batch size, which could lead to overfitting. I suggest using four GPUs for training. I will close this repository first. If there are any issues, feel free to reopen it and ask further questions.
